tests/extra/job_management/test_job_management.py

```diff
@@ -1761,3 +1768,88 @@ def test_with_job_manager_parameter_column_map(
                 "description": "Process 'increment' (namespace https://remote.test/increment.json) with {'data': 5, 'increment': 200}",
             },
         }
+
+class TestJobManagerWorkerThread:
+
+    @pytest.fixture
+    def worker(self):
+        work_queue = queue.Queue()
+        result_queue = queue.Queue()
+        return _JobManagerWorkerThread(work_queue, result_queue)
+
+    def test_successful_job_start(self, worker):
+        # Setup
+        mock_conn = mock.MagicMock()
+        mock_job = mock.MagicMock()
+        mock_job.start.return_value = None
+        mock_job.status.return_value = "running"
+        mock_conn.job.return_value = mock_job
```

Comment:
Isn't it possible to use `requests_mock` to mock the openEO API instead of mocking our own internal implementation (which risks not testing what we actually want to test) ?